db/migrate/20160303115326_add_websocket_to_miq_server.rb

```diff
@@ -0,0 +1,5 @@
+class AddWebsocketToMiqServer < ActiveRecord::Migration[5.0]
+  def change
+    add_column :miq_servers, :has_active_websocket, :boolean
```

Comment:
I expect a lot of follow up work. More console types for more providers and the proxy stuff that we discusse.
So I'd prefer to get this stuff merged so that we can support consoles in the SSUI fo the next release and do further work in other PRs.